cassovary-core/src/main/scala/com/twitter/cassovary/graph/MemoryMappedDirectedGraph.scala

```diff
@@ -0,0 +1,153 @@
+package com.twitter.cassovary.graph
```

Comment:
I'd suggest removing the copyright statement altogether -- that's what I am doing for any files that are new altogether.